app/(frontend)/privacy-policy/page.tsx

```diff
@@ -0,0 +1,594 @@
+      otherwise. This includes personal data you provide when you:
+    </p>
+    <ol>
+      <li>a) present Website content;</li>
+      <li>b) use any of our Services;</li>
+      <li>c) create an account on our Website;</li>
+      <li>d) subscribe to our Services or publications;</li>
+      <li>e) request marketing to be sent to you;</li>
+      <li>f) give us some feedback.</li>
+    </ol>
+    <p>
+      4.2 In addition to the above, we may use the information in the following
+      ways:
+    </p>
+    <ol>
+      <li>
+        a) To personalise your Website experience and to allow us to deliver the
+        type of content and product offerings in which you are most interested.
+      </li>
+      <li>
+        b) If you have consented to receive our e-mail newsletter, we may send
+        you periodic e-mails. If you would no longer like to receive promotional
+        e-mail from us, please refer to the "How can you opt-out, remove or
+        modify information you have provided to us?" section below. If you have
+        not opted-in to receive e-mail newsletters, you will not receive these
+        e-mails. Visitors who register or participate in other site features
+        such as marketing programs and 'members-only' content will be given a
+        choice whether they would like to be on our e-mail list and receive
+        e-mail communications from us.
+      </li>
+      <li>c) Present Website content effectively to you.</li>
+      <li>
+        d) Provide information, and services that you request, or (with your
+        consent) which we think may interest you.
+      </li>
+      <li>d) Carry out our contracts with you.</li>
+      <li>e) Provide the relevant Services to you</li>
+      <li>f) Tell you our charges.</li>
+    </ol>
+    <p>
+      4.3 If you are already our customer, we will only contact you
+      electronically about things similar to what was previously sold to you.
+    </p>
+    <p>
+      4.4 If you are a new customer, you will only be contacted if you agree to
+      it.
+    </p>
+    <p>
+      4.5 We may keep a record of those links which are used the most to enable
+      us to provide the most helpful information but we agree to keep such
+      information confidential and you will not be identified from this
+      information.
+    </p>
+    <p>
+      4.6 In addition, if you don't want us to use your personal data for any of
+      the other reasons set out in this section in 4, you can let us know at any
+      time by contacting us at
+      <em />{" "}
+      <Link href="mailto:privacy@frostwork.io">privacy@frostwork.io</Link>, and
+      we will delete your data from our systems. However, you acknowledge this
+      will limit our ability to provide the best possible services to you.
+    </p>
+    <p>
+      4.7 In some cases, the collection of personal data may be a statutory or
+      contractual requirement, and we will be limited in the{" "}
+      <strong>
```

Comment:
`<em><strong>[</strong>products and <strong>]</strong></em>`?

Reply:
I used a tool to convert the google docs to html. Lemme fix that.